docs/release-notes/release-notes-0.7.1.md

```diff
@@ -0,0 +1,117 @@
+
+`23d0b8f` BIP78 receiver over a Tor hidden service.
+
+`2b898be` Fix finalization check of PSBT Inputs
+
+`2da073b` Bump python-bitcointx to v1.1.1.post0
+
+`9886553` Adds close button to BIP78 receiver dialog
+
+### Transaction broadcast via peers
+
+A long standing issue since Joinmarket-clientserver was created as an alternative implementation: the original version of Joinmarket allowed participants to request transaction broadcast over the message channel, instead of broadcasting it themselves, for further obfuscation. But part of the code required for this was not implemented. This update corrects that, and now both `random-peer` and `not-self` broadcast options (set in `[POLICY]` `tx_broadcast` config option), as documented in the config file, are allowed again.
+Note, however, that while many bots are not updated to this version, very often your bot *will* fall back to broadcasting the transaction itself, because the bot you requested this action from simply ignores the request in the old code. This is noted in the config comments.
+
+This patch was prompted because a recent update actually resulted in choosing this option was causing a crash rather than a broadcast failure.
+
+`d209d4d` Fixes #368. random-peer and not-self tx broadcast.
+
+`4440ffb` Taker broadcasts tx after unconfirm_timeout_sec.
+
+#### Add wallet generation script.
+
+This is not recommended for general users (care needs to be taken over passwords and seeds), but a script `genwallet.py` is added to allow non-manual creation of a wallet for those people scripting automation tools.
+
+`93f0a67` `2f53c5f` Add genwallet.py script, add create_wallet tests
+
+#### Ob-watcher improvements
+
+Running ob-watcher locally should be simple enough since it doesn't require Bitcoin Core, but we no longer require installation of matplotlib either (which is a rather large python dependency). Also some untidy number formatting in the main orderbook table is cleaned up.
+
+`6d62661` Allow running ob-watcher without matplotlib installed
+
+`942ea58` Fix relfee cjfee display issues caused by the use of float
+
+##### Various improvements and bugfixes
+
+Of note here: command line now shows UTXO account/mixdepth info (as Qt already did).
+
+`d946009` add mixdepth information in the list of utxos returned by the showutxos command
+
+`0e5eb2f` More verbose absurd fee exception message
+
+`5c85a3d` Log sendrawtransaction errors as warning not debug
+
+`8d63cb8` correct sendpayment help message for -N
+
+`2076dbf` Improve checks for send ("Single Join") in GUI
+
+`5af2d49` handle Qt wallet load failure
+
+`a2aafd2` `202f8ee` `5604857` Fixes #673. Shutdown cleanly on failure to access blockheight
```

Comment:
```suggestion
`a2aafd2` `202f8ee` `5604857` Fixes [#673](https://github.com/JoinMarket-Org/joinmarket-clientserver/issues/673). Shutdown cleanly on failure to access blockheight
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/701/commits/cdccb958b67bd7aed146236ea688760c96531a37